src/main/resources/static/style.css

```diff
@@ -99,10 +99,10 @@ a:hover {
 
 #audioPlayer {
     display: none;
-    /*initially hide the audioplayer*/
+    /* Initially hide the audioplayer*/
```

Comment:
Add a space at the end. 
`/* Initially hide the audioplayer */`

Same for other similar comments.